src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -34,7 +40,28 @@
   &.pf-m-lg {
     max-width: var(--pf-c-empty-state--m-lg--MaxWidth);
   }
+
+  &.pf-m-xl {
+    > .pf-c-button.pf-m-primary,
```

Comment:
@mcoker @mattnolting is there a better way to write this selector

Reply:
Yeah I think so, I submitted a PR if you want to take a look and compare https://github.com/christiemolloy/patternfly-next/pull/19